articles/java-jdbc/index.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/java-jdbc/
+title: Getting Started with Java JDBC
+description: This article will be an introduction to Java database connectivity, we will learn how to perform various database query operations from a Java application.
+author: paul-juma
+date: 2021-01-22T00:00:00-17:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/java-jdbc/hero.jpg
+    alt: Java database connectivity (JDBC) example
+---
+Java database connectivity (JDBC) is an application programming interface that defines how Java application programs connect and execute database queries.
+<!--more-->
+### Prerequisites
```

Comment:
Please provide some further introduction and goal of your project just before `Prerequisites`